onnxscript/function_libs/torch_aten/param_manipulation.py

```diff
@@ -0,0 +1,46 @@
+"""Function for manipulating input parameters of an Op or a OnnxFunction."""
+from __future__ import annotations
+
+import collections
+import dataclasses
+from typing import Any, List, OrderedDict, Sequence, Tuple
+
+# A special value to indicate that the default value is not specified
+EmptyDefault = object()
+
+
+@dataclasses.dataclass(frozen=True)
+class ParamSchema:
```

Comment:
TODO: create two functions that takes from OnnxFunction or Op and produce a list of `ParamSchema`s